vue-components/src/components/Link.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<span
+		:class="[
+			'wikit',
+			'wikit-Link',
+			(underlined == true) ? `wikit-Link--underlined` : ''
+		]"
+	>
+		<Icon
+			v-if="icon === 'before'"
+			class="wikit-Link__icon wikit-Link__icon--before"
+			type="link"
+			color="progressive"
+			size="large"
+		/>
+		<span class="wikit-Link__content"><slot /></span>
+		<Icon
+			v-if="icon === 'after'"
+			class="wikit-Link__icon wikit-Link__icon--after"
+			type="newwindow"
+			color="progressive"
+			size="large"
+		/>
+	</span>
```

Comment:
maybe the Icon element could just be one. 
something like this, maybe
`<Icon
        class="wikit-Link__icon"
        :class="[ `wikit-Popover--${icon}` ]"
        v-if="icon !== 'none'"
        type="setLink()"
        color="progressive"
         size="large"
/>`
and you can set the type based on the `Icon` type...

its just a thought. doesn't have to be implemented.

Reply:
Thanks but I don't understand how that place the icon before and after the slot. Am I missing something obvious?

Reply:
you're right. I missed the icons